week04/Search_Suggestions_System.cpp

```diff
@@ -0,0 +1,52 @@
+// TC: O(N*S) N is the length of searchWord string
+// MC: O(S) S is sum of all strings at products
```

Comment:
TC analysis is wrong. The actual TC is `O(N * 3 * L)` where L = max length of a string in `products`.

`3L` is much smaller than `S`.